io.catenax.shared.recycling_strategy_certificate/2.0.0/RecyclingStrategyCertificate.ttl

```diff
@@ -0,0 +1,108 @@
+#######################################################################
+# Copyright (c) 2023 SAP SE
+# Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft
+# Copyright (c) 2023 LRP Autorecycling Leipzig GmbH
+# Copyright (c) 2023 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
+# Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.shared.recycling_strategy_certificate:2.0.0#> .
+
+:RecyclingStrategyCertificate a samm:Aspect ;
+   samm:preferredName "Recycling strategy certificate"@en ;
+   samm:description "The certificate marks the point in time at which an asset irrevocably enters a new life. The eol (end of life) phase is completed and a new product life cycle is started."@en ;
+   samm:properties ( :recyclingStrategyCertificateProperty ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:recyclingStrategyCertificateProperty a samm:Property ;
+   samm:preferredName "Recycling strategy certificate property"@en ;
+   samm:characteristic :RecyclingStrategyCertificateCharacteristic .
+
+:RecyclingStrategyCertificateCharacteristic a samm:Characteristic ;
+   samm:preferredName "Recycling strategy certificate characteristic"@en ;
+   samm:dataType :RecyclingStrategyCertificateEntity .
+
+:RecyclingStrategyCertificateEntity a samm:Entity ;
+   samm:preferredName "Recycling Strategy Certificate"@en ;
+   samm:properties ( :issuer :issueDate :catenaXIdPreviousLife :catenaXId :articleNumber :vin :oeNumber ) .
+
+:issuer a samm:Property ;
+   samm:preferredName "Issuer"@en ;
+   samm:description "BPMN of the business partner who creates this certificate"@en ;
+   samm:characteristic :IssuerCharacteristic ;
```

Comment:
You can make use of existing characteristic and don't need to define a new one.
For reference: https://github.com/eclipse-tractusx/sldt-semantic-models/pull/405/files#diff-8294717bc9be9d276f548ae9f83e2831503589ef17d4d5bf6f1463cdb397d00eR104

Reply:
The naming comes from the feature team. I don't know if "supplier" is suitable according to their opinion. I will ask them.

Reply:
So here I am not referring to change the name, but to use the shared aspect model for BPNL characteristic. You will see in the referenced link that they have not defined a new characteristic (like you did IssuerCharacteristic), so you directly put ext-number:BpnlTrait  instead 'IssuerCharacteristic' in your model, providing you define '@prefix ext-number: <urn:samm:io.catenax.shared.business_partner_number:1.0.0#> .' at the top of model.
Let me know if there is still any confusion.